frontend/actions/accessRequest.ts

```diff
@@ -49,3 +49,11 @@ export function patchAccessRequest(modelId: string, accessRequestId: string, for
     body: JSON.stringify({ metadata: { ...form } }),
   })
 }
+
+export function postAccessRequestComments(modelId: string, accessRequestId: string, comment: string) {
```

Comment:
This only allows one comment to be sent, consider renaming to `postAccessRequestComment`